### PR TITLE
Add custom request limits per domain and publish alpha 76

### PR DESCRIPTION
**Pull request by tephenavies**
### What this PR does

Allows setting custom request limits per domain. This was trialled in https://github.com/TerriaJS/de-australia-map and worked well, however using a limit of 64 is probably too high and will interfere with Cesium's request prioritisation. Values from 12-24 should be the sweet spot for significantly improving performance where parallel requests are the limiting factor.

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist)
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Review by nf-s — CHANGES_REQUESTED** [review]
Can you please add new properties to `doc\customizing\client-side-config.md`

File: doc/customizing/client-side-config.md [hunk 1]
```diff
@@ -78,6 +78,9 @@ Specifies various options for configuring TerriaJS:
 |`helpContent`|no|**[HelpContentItem](#HelpContentItem)**|`[]`|The content to be displayed in the help panel.|
 |`helpContentTerms`|no|**[Term](#Term)**|||
 |`languageConfiguration`|no|**[LanguageConfiguration](#LanguageConfiguration)**|undefined|Language configuration of TerriaJS.|
+|`customRequestSchedulerLimits`|no|**object**|undefined|Custom concurrent request limits for domains in Cesium's RequestScheduler.|
```
Comment (nf-s) [inline doc/customizing/client-side-config.md]:
```suggestion
|`customRequestSchedulerLimits`|no|**[RequestScheduler](https://cesium.com/docs/cesiumjs-ref-doc/RequestScheduler.html)**|undefined|Custom concurrent request limits for domains in Cesium's RequestScheduler.|
```

**Comment by tephenavies** [comment]
This PR will publish alpha 76

**Review by nf-s — CHANGES_REQUESTED** [review]

File: doc/customizing/client-side-config.md (hunk 1 above)
Comment (tephenavies) [reply doc/customizing/client-side-config.md]:
```suggestion
|`customRequestSchedulerLimits`|no|**[RequestScheduler](https://cesium.com/docs/cesiumjs-ref-doc/RequestScheduler.html#.requestsByServer)**|undefined|Custom concurrent request limits for domains in Cesium's RequestScheduler.|
```

**Review by nf-s — APPROVED** [review]

**Pull request merged**